src/code/providers/google-drive-provider.ts

```diff
@@ -278,7 +307,9 @@ class GoogleDriveProvider extends ProviderInterface {
     })
   }
 
-  close(metadata: CloudMetadata, callback: ProviderCloseCallback) {}
+  close(metadata: CloudMetadata, callback: ProviderCloseCallback) {
+    console.log('Closing GD File');
```

Comment:
Seems like this could get noisy as well. Was this for debugging or were you proposing keeping it moving forward?

Reply:
It was for debugging. Removing.